packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -376,31 +378,76 @@ export const components: ThemeOptions["components"] = {
   MuiCheckbox: {
     defaultProps: {
       size: "small",
+      icon: <></>,
+      checkedIcon: <CheckIcon />,
+      indeterminateIcon: <SubtractIcon />,
     },
     styleOverrides: {
       root: ({ theme }) => ({
-        borderRadius: "4px",
-        "&:hover": {
+        width: theme.spacing(4),
```

Comment:
```suggestion
        width: `${theme.typography.body1.lineHeight}em`,
```

Something like this for `width` and `height` would be less brittle. Like this, the box size is always associated with the label typography instead of our global spacing units.